settings/src/components/webauthn/webauthn.js

```diff
@@ -128,28 +113,85 @@ export default function WebAuthn() {
 
 			<p className="wporg-2fa__submit-actions">
 				<Button variant="primary" onClick={ () => setFlow( 'register' ) }>
-					Register New Key
+					Register new key
 				</Button>
 
 				{ keys.length > 0 && (
 					<Button
 						variant="secondary"
-						onClick={ webAuthnEnabled ? disableProvider : enableProvider }
+						onClick={ webAuthnEnabled ? showConfirmDisableModal : toggleProvider }
 						disabled={ statusWaiting }
 					>
-						{ webAuthnEnabled ? 'Disable Security Keys' : 'Enable Security Keys' }
+						{ `${ webAuthnEnabled ? 'Disable' : 'Enable' } security keys` }
 					</Button>
 				) }
-
-				{ statusWaiting && <Spinner /> }
 			</p>
 
+			{ statusWaiting && (
+				<p className="wporg-2fa__webauthn-register-key-status">
+					<Spinner />
+				</p>
+			) }
+
 			{ statusError && (
```

Comment:
I don't know if this is related to this PR, but when my session expires I kind of get stuck not knowing what to do. We should probably show the revalidate session before I even get this far.

<img src="https://github.com/WordPress/wporg-two-factor/assets/1657336/76de81cd-e50b-476d-a0d6-3ec525a983fa" width="300" />

Reply:
Yeah let's address this separately

Reply:
New issue: https://github.com/WordPress/wporg-two-factor/issues/224